wormhole-connect/src/config/tokens.ts

```diff
@@ -378,10 +405,9 @@ export class TokenCache extends TokenMapping<Token> {
 
         mapping.setLocalStorageKey(localStorageKey);
 
-        for (const [key, tokenData] of Object.entries(asJson.tokens)) {
-          const tokenId = parseTokenKey(key);
+        for (const [_key, tokenData] of Object.entries(asJson.tokens)) {
```

Comment:
nit: can do `[, tokenData]` to get rid of the unused `_key` warning

Reply:
https://github.com/wormhole-foundation/wormhole-connect/pull/3200/commits/3ccc38750c4f31187953be6d266e21e70867c157